readme.md

```diff
@@ -110,13 +119,16 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 | Query enrollments                                  | :heavy_check_mark: | Programmatically query registration states with the service SDK.  Please visit the [samples folder](./provisioning/service/samples/) to learn more about this feature.                                                                            |
 
 ## Read more
-* [Azure IoT Hub documentation][iot-hub-documentation]
-* [Set up IoT Hub](doc/setup_iothub.md) describes how to configure your Azure IoT Hub service.
-* [Manage IoT Hub](doc/manage_iot_hub.md) describes how to provision devices in your Azure IoT Hub service.
-* [Azure Certified for IoT device catalog](https://catalog.azureiotsuite.com/)
-* [Set up your development environment](./doc/devbox_setup.md) to prepare your development environment as well as how to run the samples on Linux, Windows or other platforms.
-* [API reference documentation for .NET](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/devices?view=azure-dotnet)
-* [Get Started with IoT Hub using .NET](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-csharp-csharp-getstarted)
+
+- [Azure IoT Hub documentation][iot-hub-documentation]
+- [Set up IoT Hub](doc/setup_iothub.md) describes how to configure your Azure IoT Hub service.
+- [Manage IoT Hub](doc/manage_iot_hub.md) describes how to provision devices in your Azure IoT Hub service.
+- [Azure Certified for IoT device catalog](https://catalog.azureiotsuite.com/)
+- [Set up your development environment](./doc/devbox_setup.md) to prepare your development environment as well as how to run the samples on Linux, Windows or other platforms.
+- [API reference documentation for .NET](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/devices?view=azure-dotnet)
+- [Get Started with IoT Hub using .NET](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-csharp-csharp-getstarted)
+
+> Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
```

Comment:
Can add a note that we can pick the old Device Explorer from Releases tab (also Azure IoT Explorer doesn't seem to support the ability to send C2D messages) (??)